io.catenax.shared.recycling_strategy_certificate/RELEASE_NOTES.md

```diff
@@ -3,9 +3,27 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [3.0.0] - 2024-02-28
```

Comment:
Now you can change the date to 4th March if you want MS3 in next meeting on Monday